example-consuming-client/.gitignore

```diff
@@ -0,0 +1,4 @@
+/node_modules
```

Comment:
Do we need 2nd gitignore?

Reply:
Good question -- And no... but i am potentially going to be ripping this demo app out and putting it with an api that will be swarmed.
Leaving now since it has not affect on lib.